slides/main.md

```diff
@@ -67,7 +67,7 @@ header-includes:
 
 4. Run an example:
 
-    ```bash
+    ```plain
```

Comment:
you are right. this looks better

Reply:
Yes. Sadly the `bash` formatting did odd things with reserved keywords like `env` and `config`